extensions/camel/camel-servlet/runtime/src/main/java/io/quarkus/camel/servlet/runtime/CamelServletConfig.java

```diff
@@ -3,8 +3,6 @@
 import java.util.List;
 import java.util.Map;
 
-import javax.servlet.Servlet;
```

Comment:
The changes in this file were applied automatically during the build process and ci fails without them

Reply:
I just merged another PR containing the same change.

Reply:
OK thanks, I will rebase then